tests/std/tests/P0226R1_math_special_functions/test.cpp

```diff
@@ -0,0 +1,542 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <limits>
+#include <utility>
+
+using namespace std;
+
+template <class Float>
+bool isclose(Float f, Float g, const int ulps = 1) {
+    if (f == g) {
+        return true;
+    }
+
+    if (f > g) {
+        swap(f, g);
+    }
+    // f < g
+    for (int i = 0; i < ulps; ++i) {
+        f = nextafter(f, numeric_limits<Float>::infinity());
+        if (f == g) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+template <class Float>
+constexpr static Float half_pi = static_cast<Float>(1.57'079'632'679'489'662);

```

Comment:
We conventionally say `static constexpr`, but really this should just be plain `constexpr` like the following ones.